kyuubi-server/src/main/resources/sql/derby/statestore-schema.derby.sql

```diff
@@ -0,0 +1,47 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one or more
+-- contributor license agreements.  See the NOTICE file distributed with
+-- this work for additional information regarding copyright ownership.
+-- The ASF licenses this file to You under the Apache License, Version 2.0
+-- (the "License"); you may not use this file except in compliance with
+-- the License.  You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+--
+
+CREATE TABLE batch_metadata(
+    key_id bigint PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    batch_id varchar(36) NOT NULL,
+    batch_owner varchar(1024) NOT NULL,
+    ip_address varchar(512),
+    session_conf clob,
```

Comment:
what's the difference between conf and session conf?
Do both of them need external storage to restore?

Reply:
Before I used two tables to save `BatchState` and `BatchMeta`.

Have discussed with @ulysses-you , we decided to simplify it with one table.

Reply:
It seems not the answer of my question

Reply:
I think the answer is: 
- conf is a request conf of batch parameter, which is same with jdbc url in thrift mode
- session conf is a snapshot of the kyuubi conf for that user, which can be defined at kyuubi server side

so the final conf to recover(if need) a batch should be:
session conf + request conf + advisor conf (if exists)

Reply:
session conf seems unnecessary to me

Reply:
Just in case the conf at server side changed after kyuubi instance restart. I'm fine to remove the session conf in state store and that means we should always use the current kyuubi instance conf.

Reply:
if kyuubi instance runs on Kubernetes， the old instance is very likely never come back when a new pod is scheduled， and the legacy conf might produce some uncertain things